pkgs/applications/virtualization/docker/default.nix

```
@@ -197,9 +197,9 @@ rec {
   };
 
   docker_19_03 = makeOverridable dockerGen {
-    version = "19.03.11";
-    rev = "42e35e61f352e527082521280d5ea3761f0dee50";
-    sha256 = "0q4l4z2bg269hj1dqaq02f84zgp3p8idw1cxczw383cjl0ra8xgq";
+    version = "19.03.12";
+    rev = "48a66213fe1747e8873f849862ff3fb981899fc6";
```

are we not using releases? https://github.com/docker/docker-ce/releases
```suggestion
    rev = "v${version}";
```
would also have to make it `rec`

---

That makes sense. I'll try it tonight.